packages/config/src/bin/serve-chrome.ts

```diff
@@ -0,0 +1,164 @@
+import axios from 'axios';
+import path from 'path';
+import fs from 'fs';
+import { execSync, spawn } from 'child_process';
+import { load } from 'js-yaml';
+import { LogType, fecLogger } from '@redhat-cloud-services/frontend-components-config-utilities';
+import waitOn from 'wait-on';
+
+const REPO_OWNER = 'RedHatInsights';
+const REPO_NAME = 'insights-chrome';
+const CONTAINER_PORT = 8000;
+export const EXPOSED_PORT = 9999;
+
+type ContainerRuntime = 'docker' | 'podman';
+let execBin: ContainerRuntime | undefined = undefined;
+
+const chromeDeploymentConfig = {
+  repo: 'git@gitlab.cee.redhat.com:service/app-interface.git',
+  deployFile: 'data/services/insights/frontend-base/deploy.yml',
+  stablePath: 'resourceTemplates[0].targets[4].ref',
+  tarTarget: path.resolve(__dirname, 'chrome-deploy/'),
+};
+
+const checkoutCommand = `git archive --remote=${chromeDeploymentConfig.repo}  HEAD ${chromeDeploymentConfig.deployFile} |  tar xvf - -C ${chromeDeploymentConfig.tarTarget}`;
```

Comment:
Can this be simplified to pull the respective file from GitLab or directly from GraphQL of AppSRE? Ideally this could be done the same way as bonfire is pulling the configs to have it consistent.

Reply:
Is it simpler to use the AppSRE API? I am not sure. We just need one file.

Reply:
The key here is consistency and maintainability. Although, I see that it might seem like an overkill to use an API. The API can bring the consistency, as that is what AppSRE and bonfire uses.

Reply:
@vkrizan  Do we need some API keys or other creds to access the AppSRE API? If not let's switch then.

Reply:
I think it should be opened when logged in on VPN. Search for "Querying the App-interface" on app-interface repo's README.